lanl/docker-compose/generate-creds.sh

```diff
@@ -4,4 +4,5 @@
 echo "POSTGRES_PASSWORD=$(openssl rand -base64 32)" > .env
 echo "BSS_POSTGRES_PASSWORD=$(openssl rand -base64 32)" >> .env
 echo "SMD_POSTGRES_PASSWORD=$(openssl rand -base64 32)" >> .env
-echo "HYDRA_POSTGRES_PASSWORD=$(openssl rand -base64 32)" >> .env
+echo "HYDRA_POSTGRES_PASSWORD=hydra" >> .env
+echo "KRATOS_POSTGRES_PASSWORD=kratos" >> .env
```

Comment:
Randomized passwords should now work with the merge of https://github.com/OpenCHAMI/deployment-recipes/pull/24 if you want to use them.. The command line is now:

```
openssl rand -base64 32 | openssl dgst | cut -d' ' -f2
```

Reply:
Yes, I'll change that to use that command.